pylint/checkers/strings.py

```diff
@@ -244,7 +244,6 @@ class StringFormatChecker(BaseChecker):
         "format-needs-mapping",
         "too-many-format-args",
         "too-few-format-args",
-        "bad-string-format-type",
```

Comment:
To make up for the times a message is forgotten in ``check_messages`` we had ``bad-string-format-type`` twice here. 😀